camel-amq-fakeapp/README.adoc

```diff
@@ -35,8 +35,15 @@ Failure injection may be added with a future version.
 The container image can easily be build with the fabric8 build target, which uses the S2I strategy provided you are connected to OpenShift. Once you have cloned the git repository execute the following commands:
 
  $ cd auto-perf-test/camel-amq-fakeapp
+ Change amqp.host in the following file
+ camel-amq-fakeapp/src/main/openshift/camel-amq-fakeapp-props-cm.yaml
+ camel-amq-fakeapp/src/main/resources/application.properties
+
+ Create certificate as described below in section as prerequisite.
+ make sure amqp.options.transport.trustStoreLocation is correct camel-amq-fakeapp/src/main/resources/application.properties
+
  $ oc login
- $ mvn fabric8:build
+ $ mvn package fabric8:build
```

Comment:
fabric8 does an S2I build. IIRC it takes the code from the git repo

Reply:
I think it does the binary build.if you check the camel-amq-fakeapp-bc.yaml there is no mention of the git repo and it specifies  type as binary.